docs/src/guides/general/deploy-button.md

```diff
@@ -18,9 +20,9 @@ or anywhere else a link can go to make launching your code base as straightforwa
 
 To have a deployable template, you need to first prepare the repository.
 The Deploy on Platform.sh button works with any Git repository that's deployable on Platform.sh.
-It needs [app configuration](../configuration/app/_index.md),
-[a `.platform/services.yaml` file](../configuration/services/_index.md),
-and [a `.platform/routes.yaml` file](../configuration/routes/_index.md).
+It needs [app configuration](../../configuration/app/_index.md),
```

Comment:
A `.platform.app.yaml` file isn't strictly necessary. It could be an `.platform/applications.yaml` file. I think that's why it was like "app configuration" with a link to a further explanation with details of anyone needs it.